src/admin/components/elements/ColumnSelector/index.tsx

```diff
@@ -1,62 +1,115 @@
-import React, { useEffect, useId, useState } from 'react';
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useCallback, useEffect, useId, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import flattenTopLevelFields from '../../../../utilities/flattenTopLevelFields';
 import Pill from '../Pill';
 import Plus from '../../icons/Plus';
 import X from '../../icons/X';
+import DragHandle from '../../icons/Drag';
 import { Props } from './types';
 import { getTranslation } from '../../../../utilities/getTranslation';
 import { useEditDepth } from '../../utilities/EditDepth';
 import './index.scss';
+import { Field } from '../../../../fields/config/types';
```

Comment:
@phillipmalboeuf do you mind bumping these above the scss import leaving a blank line between them? Just to keep consistency with other files. (I know it likely happened when you auto imported, happens all the time)